src/playground-code-editor.ts

```diff
@@ -677,36 +677,38 @@ export class PlaygroundCodeEditor extends LitElement {
   ): TemplateResult | string {
     const markedObjectName = objectName ?? '';
     const matches = completionData?.matches ?? [];
-    let padding = 0;
     if (matches.length <= 0) {
       // In the situation, that none of the input matches with the
       // completion item suggestion, we exit early, leaving the objectName unmarked.
       return markedObjectName;
     }
 
+    let padding = 0;
```

Comment:
I wasn't able to repro or understand when this is used. In examples that I tried, length of `matches` always seems to be `1`. Do we need the `padding` logic?